lib/v2/ptu/radar.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    'ptu.edu.cn': {
+        _name: '莆田学院新闻网',
+        news: [
+            {
+                title: '莆院要闻',
+                docs: 'https://docs.rsshub.app/shopping.html#pu-tian-xue-yuan',
+                source: ['/:type/:pageId', '/'],
+                target: '/ptu/news/:type/:pageId',
+            },
+            {
+                title: '莆院快讯',
+                docs: 'https://docs.rsshub.app/shopping.html#pu-tian-xue-yuan',
+                source: ['/:type/:pageId', '/'],
+                target: '/ptu/news/:type/:pageId',
+            },
+            {
+                title: '媒体莆院',
+                docs: 'https://docs.rsshub.app/university.html#pu-tian-xue-yuan',
+                source: ['/:type/:id', '/'],
+                target: '/ptu/news/:type/:pageId',
+            },
```

Comment:
You can use a single rule to match them all instead
```js
                source: ['/:type'],
                target: (params) => `/ptu/news/${params.type.replace('.htm', '')}`,
```

Reply:
Thanks for the review, but I still don't quite understand how to improve these codes? Can you give a concrete example?

Reply:
https://docs.rsshub.app/joinus/new-radar.html